tests/baselines/reference/outModuleConcatUmd.js

```diff
@@ -17,6 +17,41 @@ var __extends = (this && this.__extends) || function (d, b) {
     d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
 };
 // This should error
+(function (dependencies, factory) {

```

Comment:
This is another case where I'd rather we not accept this file as it uses `outFile`.